Source/JavaScriptCore/runtime/MicrotaskQueue.h

```diff
@@ -90,9 +90,17 @@ class QueuedTask {
 
 class MicrotaskDispatcher : public RefCounted<MicrotaskDispatcher> {
 public:
+    MicrotaskDispatcher(unsigned typeId)
```

Comment:
Can we use enum class instead of unsigned?
i.e. move enum class Type from WebCore::WebCoreMicrotaskDispatcher to JSC:: MicrotaskDispatcher?

Reply:
I was a bit feeling it is reversed dependency but for now, let's just do it :)

Reply:
I think it's okay. `Node`, for example, has a bunch of flags for `Element`, etc... for better bitfield packing.

Reply:
It's what we do for JSType too in WebCore.